### ISSUE TITLE
Request: for media scraping: turn off GIF

### ISSUE THREAD

**Issue by DB9-max** [prinsss/twitter-web-exporter#58]
For media scraping modules suggestion: turn off GIF. 

GIFs are spam and usually useless and take memory a lot.

Put why not also: turn on/off images, turn on/off videos. 
^ If user just wants videos or images and not GIFs ;)

To be used in user media, search results, retweets, likes, bookmarks 

:)

Also saves time and effort to not include something in searching by the script if user not looking for them

**Comment by prinsss** [comment]
Yes, I agree with you. We should have filtering options in the table view so that users can filter out which tweets they want to export.

Will add this feature in the next update. Thanks for the suggestion! :)

**Comment by prinsss** [comment]
With [v1.2.1](https://github.com/prinsss/twitter-web-exporter/releases/download/v1.2.1/twitter-web-exporter.user.js), you can now set filters for which type of media to export. :)

![Image](https://github.com/user-attachments/assets/766b79ed-d049-472e-91b8-6693b2821911)
